vagrant-spk

```diff
@@ -640,22 +644,30 @@ def publish(args):
     finally:
         os.remove(temp_spk)
 
-def halt(args):
+def warn_deprecated(command_name):
+    sys.stderr.write("WARNING: `vagrant-spk {0}` is deprecated and may be removed in the future.\nPrefer `vagrant-spk vm {0}`.\n".format(command_name))
```

Comment:
It'd also be nice to link to the email thread, or github issue, in this deprecation message, so people can easily access the context for the change.

https://groups.google.com/forum/#!topic/sandstorm-dev/cuSNJ3IsP6I is a link to the message in case that saves you the trouble of finding the link.